src/components/AppointmentConfigModal.vue

```diff
@@ -216,26 +217,33 @@ export default {
 		},
 		calendar() {
 			if (!this.editing.targetCalendarUri) {
-				return this.ownSortedCalendars[0]
+				return this.sortedCalendars[0]
```

Comment:
```suggestion
				return this.availableCalendars[0]
```

This should also be affected by the feature gate, right?

Reply:
Done.

Those do not need to be feature gated. But I changed them to use the feature gate, so the logic is easier to understand. The end result is the same.